packages/hardhat-core/src/internal/util/download.ts

```diff
@@ -1,14 +1,8 @@
 import fs from "fs";
 import fsExtra from "fs-extra";
-import HttpsProxyAgent from "https-proxy-agent";
```

Comment:
Shouldn't we remove this dependency from the package.json?

Reply:
Addressed in https://github.com/NomicFoundation/hardhat/pull/2370/commits/9ae76438d6f97ebcd43e0576d1df4b05673ab755